### ISSUE TITLE
Add a way to suppress ui output

### ISSUE THREAD

**Issue by ABuffEr** [nvaccess/nvda#8669]
Hi,
as you know, in developing an add-on there may be situations where you want to call NVDA core scripts/functions. But these functions often contain ui.message or ui.reviewMessage, and if you want to avoid these voice and braille outputs you have to copy (and modify) entire function code, or elaborate another creative (and potentially dangerous, especially for Braille) solutions.
Let's take, for example, script_review_activate, in globalCommands, that is quite long (more of 3 lines). When called, it often speaks/shows "press", "click", and other similar unnecessary outputs, that may have sense when script performs directly, but that in my opinion spoil a good user experience in other situations.
So, reassuming: to avoid these outputs and simultaneously massive recoding, can it be possible at least to introduce a flag or something else in ui to modify/call before (and after if necessary) calling of core function?
Thanks.

**Comment by larry801** [comment]
As for speech, you can set it to off temporarily.
Here is an example in NVDA code.
```
oldSpeechMode=speech.speechMode
speech.speechMode=speech.speechMode_off
eventHandler.executeEvent("gainFocus",focus.parent)
speech.speechMode=oldSpeechMode
```

On Fri, Aug 24, 2018 at 2:27 PM, Alberto Buffolino <notifications@github.com

**Comment by ABuffEr** [comment]
Yes, I know. I specified voice and braille outputs primarily for this reason, I read speech and braille modules before posting, but if for speech this solution exists, what say for braille? I did not found a simple and zero-complication solution. I imagine I can set temporarily on "no Braille", of course, but then I must set current display again, with potential reconnection/delay/other problems (if not in my display, potentially on other models). So, imho, a flag or similar sounds more simple and preferable...
